lib/queryBuilder/QueryBuilder.js

```diff
@@ -130,19 +130,19 @@ class QueryBuilder extends QueryBuilderBase {
     return this;
   }
 
-  modify() {
-    if (typeof arguments[0] === 'string') {
-      const namedFilters = this._modelClass.namedFilters;
-
-      for (let i = 0, l = arguments.length; i < l; ++i) {
-        const filter = namedFilters[arguments[i]];
-        filter(this);
-      }
-
-      return this;
-    } else {
-      return super.modify.apply(this, arguments);
+  applyFilter() {
+    const namedFilters = this._modelClass.namedFilters;
+    for (let i = 0, l = arguments.length; i < l; ++i) {
+      const filter = namedFilters[arguments[i]];
+      filter(this);
     }
+    return this;
+  }
+
+  modify() {
+    return typeof arguments[0] === 'string'
+      ? this.applyFilter.apply(this, arguments)
+      : super.modify.apply(this, arguments);
```

Comment:
I don't know if you dig ternary operators or not... I do, as you can see here : )
Happy to rewrite it with if / else if you prefer.

Reply:
I also like ternary operators when they make stuff simpler. I wish ecmascript handled `if`, `try` etc. blocks as expressions that return a value like kotlin does:
```js
const foo = if (x < 10) "bar" else "baz"
const bar = try {
  doStuff();
} catch (err) {
  null
}
```

Reply:
Yeah that would be nice. CoffeeScript does this also, and I believe Ruby and a bunch of others too.